lib/entities/integrations/index.ts

```diff
@@ -19,11 +20,30 @@ export class Integrations {
    */
   public async getAll(planIdentifier: string): Promise<IIntegration[] | undefined> {
       const urlSegments = [planIdentifier, this.baseUrl];
+         return await wrapWithErrorHandler(async () => {
+            const url = buildApiUrl(urlSegments);
+            const res = await this.networkClient.get<IIntegration[]>(url);
+            return res;
+        });
+    }
 
-       return await wrapWithErrorHandler(async () => {
-          const url = buildApiUrl(urlSegments);
-          const res = await this.networkClient.get<IIntegration[]>(url);
-          return res;
-      });
+  /**
+   * Create a new integration for the given plan
+   * @param planIdentifier - Identifier of the target plan
+   * @param integration - The integration to create
+   * @returns The created integration
+   */
+  async create(planIdentifier: string, integration: ICreateIntegrationPayload): Promise<IIntegration | undefined> {
```

Comment:
Was the type undefined for the Promise for creating an alert?

Reply:
yep! so does applications, so I figured I would keep this consistent in integrations too